auth_totp/controllers/main.py

```diff
@@ -81,12 +45,12 @@ def mfa_login_post(self, *args, **kwargs):
         """Process MFA login attempt
 
         Overview:
-            * Try to find a user based on the MFA login token. If this doesn't
+            * Identify current user based on login in session. If this doesn't
```

Comment:
The `in` after `login` is superfluous. Or maybe there's a word missing?

Reply:
The sentence is referring to the ``login`` value in the ``session`` dictionary.